src/components/NavigationBar.js

```diff
@@ -11,8 +11,11 @@ import phone from '../assets/phone.png';
 /**
  * Navigation bar of the app, hide/show the add new content button.
  *
- * @param {string} display - current page that is dispalyed.
- * @param {string} setDisplay - change the dispalyed page.
+ * @typedef {object} Props
+ * @prop {string} display - current page that is dispalyed
```

Comment:
nit:
```suggestion
 * @prop {string} display - current page that is displayed
```
(also same change on next line)

Reply:
fixed. thanks!